Gemfile

```diff
@@ -63,11 +63,11 @@ group :jekyll_optional_dependencies do
   gem "toml", "~> 0.1.0"
   gem "coderay", "~> 1.1.0"
   gem "jekyll-docs", :path => '../docs' if Dir.exist?('../docs') && ENV['JEKYLL_VERSION']
-  gem "jekyll-gist", "~> 1.0"
-  gem "jekyll-feed", "~> 0.1.3"
-  gem "jekyll-coffeescript", "~> 1.0"
-  gem "jekyll-redirect-from", "~> 0.9.1"
-  gem "jekyll-paginate", "~> 1.0"
+  gem "jekyll-gist"
+  gem "jekyll-feed"
+  gem "jekyll-coffeescript"
+  gem "jekyll-redirect-from"
+  gem "jekyll-paginate"
```

Comment:
:wave: @mattr-!!

A couple of them are used in our tests but are not runtime dependencies. By pinning them, we open ourselves up to issues where we release a new version of one of these, say jekyll-paginate, and we don't catch that it is broken in these tests as we upgrade this code (e.g. we could break an API)